plugins/browser-plugin-web-vitals/src/index.ts

```diff
@@ -0,0 +1,120 @@
+import { BrowserPlugin, BrowserTracker, addEventListener } from '@snowplow/browser-tracker-core';
+import { ReportCallback } from './types';
+import { WEB_VITALS_SCHEMA } from './schemata';
+import { dispatchToTrackersInCollection } from '@snowplow/browser-tracker-core';
+import { LOG, buildSelfDescribingEvent } from '@snowplow/tracker-core';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+declare global {
+  interface Window {
+    webVitals: {
```

Comment:
Even if the `loadWebVitalsScript` is false, it does not mean that we do not have acces to the `window.webVitals` interface. It means that the client did not choose to load the script with our 'automatic way'. 
Some pages might inject it at some other point, maybe even asynchronously through other means.

In case the `window.webVitals` is not present, we consider that another issue, that is why we added the:
`LOG.warn('The window.webVitals API is currently unavailable. web_vitals events will not be collected.');`
Hope this makes sense